### PR TITLE
✨Support different animation styles in `amp-lightbox`

### LINKED ISSUE #2374: amp-lightbox animation customization
Currently an amp-lightbox appear animation is fixed to an opacity transition that lasts 0.1s We'd like to be able to customize this behaviour (for example, a slide in).

When looking at the amp-lightbox source code, there's a comment from @dvoytenko that says "use new animations support instead", but I haven't been able to find any documentation about this new animation system in the AMP source code. 

Where could I find more information about this?

### PR DESCRIPTION
**Pull request by alanorozco**
Fixes #2374.

### CODE REVIEW

File: extensions/amp-lightbox/validator-amp-lightbox.protoascii [hunk 1]
```diff
@@ -47,6 +47,12 @@ tags: {  # <amp-lightbox>
   attrs: { name: "controls" }
   attrs: { name: "from" }
   attrs: { name: "scrollable" }
+  attrs: {
+    name: "animate-in"
```
Comment (honeybadgerdontcare) [inline extensions/amp-lightbox/validator-amp-lightbox.protoascii]: please alphabetize

also, repeated fields for value and value_casei is not on github yet, but we'll try to get it there today.

Comment (alanorozco) [reply extensions/amp-lightbox/validator-amp-lightbox.protoascii]: Alphabetized.

Waiting for repeated `value_casei` support to make it into the repo and then will merge.

**Comment by cathyxz** [comment]
Actually, can you also verify that the manual test referenced in https://github.com/ampproject/amphtml/pull/14676 still works? Should be /test/manual/amp-lightbox-focus.amp.html on iOS.

File: extensions/amp-lightbox/0.1/amp-lightbox.js [hunk 2]
```diff
@@ -201,24 +245,42 @@ class AmpLightbox extends AMP.BaseElement {
...
-      st.setStyle(this.element, 'webkitOverflowScrolling', 'touch');
+      st.setStyle(element, 'webkitOverflowScrolling', 'touch');
     }
 
     // This should be in a mutateElement block, but focus on iOS won't work
     // if triggered asynchronously inside a callback.
-    st.setStyles(this.element, {
-      display: '',
-      opacity: 0,
+    st.setStyles(element, dict({
```
[the remark this answers is not in the document]
Comment (alanorozco) [reply extensions/amp-lightbox/0.1/amp-lightbox.js]: Documented.

**Comment by alanorozco** [comment]
@cathyxz 

Manual test working just fine :)

**Comment by cathyxz** [comment]
LGTM

**Comment by aghassemi** [comment]
@alanorozco thanks for taking this over FixIt week! Could you create a demo when it makes it to prod and sent it to the team for a look and feedback?

**Pull request merged**